### PR TITLE
New status page

### PR DESCRIPTION
**Pull request by suehighmoor**
How to use the new service status page

### CODE REVIEW

**Comment by suehighmoor** [comment]
I've changed the title of the status page article and added an article for using webhooks (technical review offline)

File: articles/other/other-how-use-webhooks.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  echo "<br><br>Incident Name: " . $statusObject->incident->name;
+  echo "<br>Incident Impact: " . $statusObject->incident->impact;
+  echo "<br>Incident Status: " . $statusObject->incident->status;
+  echo "<br>Created At: " . $statusObject->incident->created_at;
+  echo "<br>Updated At: " . $statusObject->incident->updated_at;
+  echo "<br>Monitoring At: " . $statusObject->incident->monitoring_at;
+
+  echo "<br><br>Incident Updates: ";
+
+  //The below foreach loop goes through each component status update
```
Comment (SteveHallUKCLoud) [inline articles/other/other-how-use-webhooks.md]: missing space? or this standard command?

File: articles/other/other-how-use-webhooks.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  foreach ($statusObject->incident->incident_updates as $indUpdate){
```
Comment (SteveHallUKCLoud) [inline articles/other/other-how-use-webhooks.md]: missing space? or this standard command?

File: articles/other/other-how-view-status-info.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+        ![Subscribe via Text Message](images/other-status-notification-text.png)
+
+    2. Click the link in the confirmation text message you receive to manage your subscriptions.
+
+4. To receive webhook notifications, click the code icon (**<>**), enter the URL of the endpoint to send the webhook to and the email address to send any associated error messages, then click **Subscribe to Notifications**.
+
+    ![Subscribe via Webhook](images/other-status-notification-webhook.png)
+
+    You'll need to set up an endpoint to receive the webhook requests. For more information about webhooks, see the Atlassian Statuspage documentation: [Webhook Notifications](https://help.statuspage.io/help/webhook-notifications). For example code for setting up an endpoint, see [*How to use webhooks to receive service status notifications*](other-how-use-webhooks.md).
```
Comment (SteveHallUKCLoud) [inline articles/other/other-how-view-status-info.md]: all one word?

File: articles/other/other-how-use-webhooks.md (hunk 1 above)
Comment (suehighmoor) [reply articles/other/other-how-use-webhooks.md]: standard command

File: articles/other/other-how-use-webhooks.md (hunk 2 above)
Comment (suehighmoor) [reply articles/other/other-how-use-webhooks.md]: standard command

File: articles/other/other-how-view-status-info.md (hunk 3 above)
Comment (suehighmoor) [reply articles/other/other-how-view-status-info.md]: Statuspage is the name of the product from Atlassian -- all one word

**Pull request merged**